requirements.txt

```
@@ -1,6 +1,7 @@
 jwt
 pyquery
 pytz
+pycups
```

It's already there: https://github.com/CompassionCH/compassion-switzerland/blob/12.0/requirements.txt#L21
```suggestion
pycups==1.9.*
```